lms/security.py

```diff
@@ -63,8 +75,8 @@ def forget(self, request):
 class LTIUserSecurityPolicy:
     """Security policy based on the information of an LTIUser."""
 
-    def __init__(self, get_lti_user: Callable[[Request], LTIUser]):
-        self._get_lti_user = get_lti_user
+    def __init__(self, get_lti_user_: Callable[[Request], LTIUser]):
```

Comment:
I'm not a Python typing expert but can `get_lti_user()` return `None`? Should this be   `Callable[[Request], Optional[LTIUser]]`?

Reply:
The aim is for it to be the non-optional one.  But because we still rely on the "legacy" `get_lti_user` that one can in theory return `None` but we assume it won't.

We can lie a bit on the typing annotation.